sdk/loadtestservice/loadtesting-rest/README.md

```diff
@@ -0,0 +1,147 @@
+# AzureLoadTesting client library for JavaScript
+Azure Load Testing provides client library in JavaScript to the user by which they can interact natively with Azure Load Testing service. Azure Load Testing is a fully managed load-testing service that enables you to generate high-scale load. The service simulates traffic for your applications, regardless of where they're hosted. Developers, testers, and quality assurance (QA) engineers can use it to optimize application performance, scalability, or capacity.
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/loadtesting)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/loadtesting?view=azure-node-preview)
+- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this package.
+
```

Comment:
You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [Azure Load Test Service Resource](https://learn.microsoft.com/azure/load-testing/).